functions/exercise2/script.js

```diff
@@ -16,8 +16,11 @@ function showResult() {
         var isPhrasePalindrome = isPalindrome(phrase);
         var message = isPhrasePalindrome ? "is a palindrome" : "isn't a palindrome";
         resultElement.value = `'${phrase}' ${message}`;
+        document.getElementById("result").classList.add("resultStyle");
```

Comment:
I wrote you about it on skype. Why do you get this element:
document.getElementById("result")
once again while you already have it?